assets/js/app.js

```diff
@@ -222,6 +222,9 @@ global.MonsieurBizRichEditorManager = class {
   initInterface() {
     this.initUiElementsInterface();
     this.initUiPanelsInterface();
+    document.dispatchEvent(new CustomEvent('monsieurBizRichEditorInitInterfaceComplete', {
```

Comment:
```suggestion
    document.dispatchEvent(new CustomEvent('mbiz:rich-editor:init-interface-complete', {
```
Because [convention seems more like this](https://gomakethings.com/custom-event-naming-conventions-in-vanilla-js)